src/contexts/PokemonDataContext.js

```diff
@@ -7,96 +8,197 @@ export const PokemonDataContext = createContext();
 export const PokemonDataProvider = ({ children }) => {
   const [allPokemons, setAllPokemons] = useState();
   const [pokemons, setPokemons] = useState();
+  const [favouritesPokemons, setFavouritesPokemons] = useState();
+
   const [currentPokemonId, setCurrentPokemonId] = useState();
   const [currentPokemon, setCurrentPokemon] = useState();
   const [refresh, setRefresh] = useState(false);
+
   const [filters, setFilters] = useState(null);
+  const [sortingValue, setSortingValue] = useState();
```

Comment:
Add some default values for the states - especially in sortingValue. Without it, the user doesn't know what is the default order:

![image](https://user-images.githubusercontent.com/43967269/117711861-e437a580-b1d3-11eb-8fad-66ba7df727d8.png)